bedrock/firefox/tests/test_base.py

```diff
@@ -756,17 +756,17 @@ def setUp(self):
     def test_fx_firstrun_40_0(self, render_mock):
         """Should use default firstrun template"""
```

Comment:
Suggestion: suggest updating this test to be more descriptive now it's no longer showing a firstrun template?
```suggestion
    def test_fx_firstrun_release_channel(self, render_mock):
        """Should redirect to /firefox/new/ page"""
```